lib/core/src/Cardano/Wallet/TokenMetadata/MockServer.hs

```diff
@@ -93,20 +111,41 @@ import qualified Data.Text.Encoding as T
 type MetadataQueryApi = "metadata" :> "query"
     :> ReqBody '[JSON] BatchRequest :> Post '[JSON] BatchResponse
 
--- | Start a metadata server.
+-- | Start a metadata server on any random port.
 --
 -- To be used with @queryServerStatic@.
 withMetadataServer
     :: IO (Server MetadataQueryApi)
     -> (TokenMetadataServer -> IO a)
     -> IO a
-withMetadataServer mkServer action = withApplication app (action . mkUrl)
+withMetadataServer = withMetadataServerExt id Nothing
+
+-- | Start a metadata server.
+withMetadataServerExt
```

Comment:
What does `Ext` mean? Seems a bit cryptic to me

Reply:
It was supposed to mean "extended". I have renamed it.